src/app/comments/comment-form-container/comment-form/comment-form.component.html

```diff
@@ -0,0 +1,97 @@
+<div>
+
+  <div *ngIf="errorMessage" class="error-message">
+    <mat-icon class="red-icon" aria-hidden="false" aria-label="Example home icon">warning</mat-icon>
+    <h2>Send failed. Network error. Please try it later!</h2>
+  </div>
+
+  <div *ngIf="successMessage" class="success-message">
```

Comment:
the name `successMessage` indicates that contains a string of a message, but used as a boolean, maybe it should be `isSendSuccessful`